jsx/StaticDataTable.js

```diff
@@ -66,6 +67,10 @@ var StaticDataTable = React.createClass({
         $("#dynamictable").DynamicTable();
       }
     }
+    if (this.props.onSort) {
```

Comment:
whoops, i meant to put this in componentDidUpdate, since that is one of the only functions where it's guaranteed that react's state is up to date. but good call, your point still applies. i'll see what i can do